component/azstorage/block_blob.go

```diff
@@ -475,8 +475,10 @@ func (bb *BlockBlob) getAttrUsingList(name string) (attr *internal.ObjAttr, err
 	var marker *string = nil
 	blobsRead := 0
 
+	maxResultsForList := bb.Config.maxResultsForList
```

Comment:
there is no need to create this variable also, we can pass down bb.config.maxResults parameter as function argument below.